### PR TITLE
css-flexbox - Tré Mitchell

### PR DESCRIPTION
**Pull request by TreMitchell**
Screenshot 2024-09-05 at 3 58 51 PM
Screenshot 2024-09-05 at 3 59 01 PM
Screenshot 2024-09-05 at 3 59 07 PM
Screenshot 2024-09-05 at 3 59 17 PM

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: calc(100% / 3);
```
Comment (shawnkost) [inline css-flexbox/align-items.html]: `flex-basis` is a child property which means it needs to be applied to flex children (an element who's direct parent has `display: flex`

File: css-flexbox/justify-content.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: 25%;
```
Comment (shawnkost) [inline css-flexbox/justify-content.html:18]: `flex-basis` is a child property which means it needs to be applied to flex children (an element who's direct parent has `display: flex`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple small changes here!

**Comment by TreMitchell** [comment]
Fixed the flex-basis issue of this assignment. The corrected code is on the other commit.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**